static/js/screenly-ose.coffee

```diff
@@ -216,8 +216,12 @@ API.View.AddAssetView = class AddAssetView extends Backbone.View
         stop: (e) ->
           (that.$ '.progress').hide()
           (that.$ '.progress .bar').css 'width', "0"
+        done: (e, data) ->
           (that.$ '.status').show()
           (that.$ '.status').html 'Upload completed.'
+          setTimeout ->
+            (that.$ '.status').fadeOut('slow')
+          , 5000
     no
```

Comment:
These edits display a message when file is success upload only. Also the message fadeOut.